src/vivarium/framework/population/manager.py

```diff
@@ -442,13 +443,16 @@ def get_population(
                     "The 'tracked' attribute pipeline should return a pd.Series but instead "
                     f"returned a {type(tracked)}."
                 )
-            idx = tracked[tracked].index
+            idx = tracked[tracked == True].index
 
         if isinstance(attributes, list):
-            # check for duplicate request:
+            # check for duplicate request
             duplicates = list(set([x for x in attributes if attributes.count(x) > 1]))
             if duplicates:
-                raise PopulationError(f"Duplicate attributes requested: {duplicates}")
+                self.logger.warning(
+                    f"Duplicate attributes requested and will be dropped: {duplicates}"
+                )
+                attributes = list(set(attributes))
```

Comment:
We've now lost determinism in the order of our attributes. Can you do this without using a set?